src/access/examples/ExampleAccessControlDelegate.cpp

```diff
@@ -125,7 +126,7 @@ class SubjectStorage
             mNode = node;
             return CHIP_NO_ERROR;
         }
-        return CHIP_ERROR_INVALID_ARGUMENT;
+        return CHIP_IM_GLOBAL_STATUS(ConstraintError);
```

Comment:
In this module, it's all business logic. We should not have IM inside of logic that can be managed internally.

This is where conversion of "failure on delegate with INVALID_ARGUMENT" means constraint error in the cluster. Otherwise we are layer crossing. Cluster layer is cluster layer. This is internal to the IM business logic in a sense